src/popup/components/AddressBook/AddressBookItem.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <BtnPlain
-    class="address-book-item"
-    :to="{ name: ROUTE_ADDRESS_BOOK_EDIT, params: { id: item.address } }"
-  >
+  <BtnPlain class="address-book-item">
```

Comment:
If there is no event watcher and no `to` prop what is the reason for using `Btn` here?